src/__tests__/components/Home.test.js

```diff
@@ -1,10 +1,54 @@
 import React from 'react';
-import { shallow } from '../../../config/enzymeConfig';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { mount, shallow } from '../../../config/enzymeConfig';
 import Home from '../../components/Home';
+import store from '../../__mocks__/store';
 
 describe('<Home />', () => {
   test('renders without crashing', () => {
-    const component = shallow(<Home />);
+    const component = mount(<Provider store={store}>
+        <MemoryRouter>
+          <Home />
+        </MemoryRouter>
+      </Provider>);
     expect(component).toHaveLength(1);
   });
+  const props = {
```

Comment:
@djallas, can we have this imported from a mock file rather than hardcoding it directly in the test file, as someone else may like to reuse this object

Reply:
Sure